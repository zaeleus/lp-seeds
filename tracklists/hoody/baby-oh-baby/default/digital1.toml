medium-ids = [
  "hoody/baby-oh-baby/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/baby-oh-baby"
duration = "3:53"

[[tracks]]
position = 2
song-id = "hoody/baby-oh-baby-cokejazz-hini-lee-remix"
duration = "3:59"
