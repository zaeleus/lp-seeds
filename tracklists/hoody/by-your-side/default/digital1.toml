medium-ids = [
  "hoody/by-your-side/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/by-your-side-feat-jinbo"
duration = "3:56"
