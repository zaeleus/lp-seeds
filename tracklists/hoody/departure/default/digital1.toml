medium-ids = [
  "hoody/departure/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/perfect-timing"
duration = "3:45"

[[tracks]]
position = 2
song-id = "hoody/geudaero-isseojwo-feat-ugly-duck"
duration = "3:16"

[[tracks]]
position = 3
song-id = "hoody/love-again"
duration = "3:34"

[[tracks]]
position = 4
song-id = "hoody/thank-you"
duration = "2:51"

[[tracks]]
position = 5
song-id = "hoody/miro"
duration = "2:43"

[[tracks]]
position = 6
song-id = "hoody/something-missing-interlude"
duration = "1:05"

[[tracks]]
position = 7
song-id = "hoody/tto"
duration = "2:52"

[[tracks]]
position = 8
song-id = "hoody/annyeonghi-adios-feat-gray"
duration = "3:21"

[[tracks]]
position = 9
song-id = "hoody/chum-feat-jclef"
duration = "4:11"

[[tracks]]
position = 10
song-id = "hoody/seongwa-ak"
duration = "3:45"

[[tracks]]
position = 11
song-id = "hoody/complex"
duration = "2:15"
