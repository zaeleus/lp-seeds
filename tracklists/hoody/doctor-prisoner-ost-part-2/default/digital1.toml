medium-ids = [
  "hoody/doctor-prisoner-ost-part-2/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/eyes"
duration = "3:26"

[[tracks]]
position = 2
song-id = "hoody/eyes-inst"
duration = "3:26"
