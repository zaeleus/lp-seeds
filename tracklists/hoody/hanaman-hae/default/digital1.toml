medium-ids = [
  "hoody/hanaman-hae/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/hanaman-hae-can-t-wait"
duration = "2:54"
