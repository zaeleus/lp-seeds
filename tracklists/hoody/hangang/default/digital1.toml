medium-ids = [
  "hoody/hangang/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/hangang"
duration = "3:20"
