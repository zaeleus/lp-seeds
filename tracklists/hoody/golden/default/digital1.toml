medium-ids = [
  "hoody/golden/default/digital1",
]

[[tracks]]
position = 1
song-id = "hoody/why-feat-george-prod-slom"
duration = "3:52"

[[tracks]]
position = 2
song-id = "hoody/golden-feat-jay-park-prod-slom"
duration = "3:40"
